sections/07_conclusions.md

```diff
@@ -52,8 +52,26 @@ addition, the stochasticity and complexity of biological systems implies that
 for some problems, for instance, predicting gene regulation in disease, perfect
 accuracy will be unattainable.
 
+We are witnessing deep learning models achieving human-level performance across
+a number of biomedical domains, and yet we do not believe that biologists and
+clinicians will be out of a job anytime soon. While deep learning methods will
+soon be (or already are) better than scientists at specific tasks, they may not
+fully grasp the bigger picture. Machine learning algorithms, including deep
+neural networks, are also prone to mistakes that humans are much less likely to
+make, such as misclassification of adversarial examples [@arxiv:1312.6199
+@arxiv:1412.6572], a reminder that these algorithms do not necessarily
+understand the semantics of the objects presented. Despite progress in
```

Comment:
Maybe, because there's arguably only one limitation described: "Despite some progress in
addressing this limitation"